drivers/timer/mcux_lptmr_timer.c

```diff
@@ -122,6 +122,10 @@ static int sys_clock_driver_init(void)
 	LPTMR_SetTimerPeriod(LPTMR_BASE, CYCLES_PER_TICK);
 	LPTMR_StartTimer(LPTMR_BASE);
 
+#if (DT_INST_PROP(0, wakeup_source))
+	/* Use wakeup-signal property if specified, else use the irq number */
+	NXP_ENABLE_WAKEUP_SIGNAL(DT_INST_PROP_OR(0, wakeup_signal, DT_INST_IRQN(0)));
```

Comment:
There is not enough value proposition being shown here to justify to be adding this property IMO. You are providing this value to a vendor specific macro. If we add this property it will become sticky and if people start using it and it turns out not to be very useful and becomes a technical debt when we decide some other way to do something, I don't want to add it. It's not thought through and seems rushed, and for what goal exactly I don't quite understand.

Reply:
Also I can see it being fulfilled by my https://github.com/zephyrproject-rtos/zephyr/pull/93838 if you think of it as a one bit multiplexor